src/main/resources/application.yml

```diff
@@ -149,6 +149,11 @@ queueconfig:
 messaging:
   publish:
     timeout: 4
+  retry:
+    initial: 1000
+    multiplier: 2
+    max: 16000
+    max-attempts: 10  
```

Comment:
Aren't these numbers pretty high? 
Before the 10th attempt we'll be waiting for 1024 seconds (approx 17 mins). 
If pubsub was down then the time to failure would be about 25 minutes, which is long enough for the caller to timeout.

Reply:
1 + 2 + 4 + 8 + 16 + 16 + 16 + 16 + 16  + 16  is about 100 seconds.
And .. there is no "caller" since we are inside a scheduled job